src/resolvers/index.test.ts

```diff
@@ -227,236 +231,326 @@ describe('GraphQL resolvers', () => {
     })
 
     describe('getMySpace', () => {
-      it('returns all widgets for the logged in user', async () => {
-        const result = await server.executeOperation({
-          query: GetMySpaceDocument,
-        })
+      type ResponseData = {
+        mySpace: MySpaceWidget[]
+      }
 
-        const expectedData = { ...newPortalUser }
+      test('returns all widgets for the logged in user', async () => {
+        const {
+          body: {
+            singleResult: { data, errors },
+          },
+        } = (await server.executeOperation<ResponseData>(
+          { query: GetMySpaceDocument },
+          { contextValue: serverContext }
```

Comment:
The context is passed in to every operation when testing with `executeOperation`, which is a bit more tedious than the last approach.